lib/view.js

```diff
@@ -267,3 +267,43 @@ export function reactVirtualizedToReactVirtualized(view) {
     , columnWidth
   }
 }
+
+/**
+  * Ensure dates and times in resources are in ISO format.
+  * @param {resource}
+  * @return {resource} with formatted dates and times
+  */
+export function normalizeDateAndTime(resource) {
+  if(resource._values) {
+    let dateFields = []
+        , timeFields = []
+        , dateTimeFields = []
+
+    resource.schema.fields.forEach((field, idx) => {
+      if(field.type === 'date') {
+        dateFields.push(idx)
+      } else if (field.type === 'time') {
+        timeFields.push(idx)
+      } else if (field.type === 'datetime') {
+        dateTimeFields.push(idx)
+      }
```

Comment:
How about using switch? Is it possible that  in the future we will have other types?